Makefile

```diff
@@ -2,3 +2,6 @@ reformat:
 	black -l 99 `git ls-files "*.py"`
 stylecheck:
 	black --check -l 99 `git ls-files "*.py"`
+gettext:
+	redgettext --command-docstrings --verbose --recursive redbot
```

Comment:
Swap with
```
redgettext --command-docstrings --verbose `git ls-files "redbot/**/*.py" --exclude "redbot/pytest/**"`
```
to avoid including pytest

Reply:
ED: This actually still includes it, one moment

Reply:
`````
redgettext --command-docstrings --verbose `git ls-files "redbot/**/*.py" | grep -v "redbot/pytest"`
`````
Works, but is ugly, the alternative is manual exclusions (from file ideally) of everything we want to exclude, with no matching pattern including wildcards, which is uglier) or a directoy exclusion option in redgettext if you are agreeable to implementing that as well.